requirements.txt

```diff
@@ -1,2 +1,3 @@
+aiosocks
```

Comment:
@Lonami 

It can be put into optional-requirements, then `import` and  `Exceptions` monkey-patching part should be moved inside
`_connect` function and `_parse_proxy` function.